src/formats/resizable-image.ts

```diff
@@ -0,0 +1,79 @@
+import { Quill } from "quill";
+
+const Image = Quill.import("formats/image");
+
+const ATTRIBUTES = ["alt", "height", "width", "class", "data-align"];
+
+/**
+ * Without this custom Quill format,
+ * image resizing will work in the editor,
+ * and it will save with the right size/alignment,
+ * but those values won't restore into the editor
+ * when you refresh or come back to the editor.
+ * This allows the `style` attribute (which is how
+ * width and height are set) to be set when Quill is
+ * loaded, properly restoring the saved image size.
+ */
+export class ResizableImage extends Image {
+    static create(props: any) {

```

Comment:
Can this be typed to anything more specific than `any`?

Reply:
The type here is basically any set of attributes that can be applied to an image. I don't think there's an interface for that. There's one in React, but not in Typescript natively.